charts/argo-cd/templates/argocd-cm.yaml

```diff
@@ -45,4 +45,7 @@ data:
 {{- if .Values.config.configManagementPlugins }}
   configManagementPlugins: |
 {{ toYaml .Values.config.configManagementPlugins | indent 4 }}
+{{ -if .Values.config.statusbadge.enabled }}
```

Comment:
As this is a boolean flag, could you please add a default to `values.yaml` and remove the `if` conditional to just use the value:
`statusbadge.enabled: "{{ .Values.config.statusbadge.enabled }}"`

Reply:
Done!